project-roles/trusted-committer.md

```diff
@@ -0,0 +1,167 @@
+# Title
+
+Trusted Comitter
+
+# Patlet
+
+Many inner-source projects will find themselves in a situation where
+they consistently receive feedback, features, and bug-fixes from contributors.
+In these situations project maintainers seek ways to recognize and reward the
+work of the contributor above and beyond single contributions.
+
+# Context
+
+  - You are the maintainer of a cross-team library, service, or shared resource
+  - You receive regular contributions
+  - There are motivated contributors looking to build expertise through
+  inner-sourced projects
+
+# Problem
+
+  - Project maintainers want to find ways to scale their ability to support a
+  project
+  - Project maintainers want to find ways to lengthen the value delivered by a
+  project
+  - Project maintainers want to visibly reward frequent contributors and
+  empower them to amplify their value contribution.
+  - Lack of language for recognizing contributions across teams within an
+  organization
+
+# Forces
+
+  - Over the lifecycle of a project the focus of the maintainers may shift away
+  to accomodate changing business priorities
+  - Contributors seek visible artifacts of their contributions demonstrating
+  value
+  - Maintaining a project of reasonable complexity is taxing for a small team
+
+# Solution
+
+## Defining the Trusted Committer Role for a Project
+
+What a Trusted Committer handles is up to each project and its maintainers.
+Whatever shape your Trusted Committer role takes make sure it's clearly
+documented somewhere in your project. This sets expectations for new community
+members and outlines the role for future candidates.
+
+Below we've provided a few guidelines for what Trusted Committers can be
+invited to do:
+
+* If a candidate participates often in community channels (e.g. Slack, JIRA
+issue triaging, etc.) then becoming a Trusted Committer formalizes their role
+in community support.
+
+* A good candidate for a Trusted Committer, is someone who frequently submits
+code, documentation, or other repository changes.  Start by including this
+person on pull requests. If they are actively engaing in pull requests,
+consider approaching them about opportunities for further collaboration on the
+project.
+
+## Formalizing Trusted Committers
+
+The first step is to approach candidates about becoming a Trusted Committer.
+Maintainers should make sure candidates understand the role. To be clear:
+there is no expectation that candidates will accept the role. Each candidate
+should figure out if they have the bandwidth to get involved.
+
+When a candidate accepts the role it is up to the project maintainers to
+publicly recognize the transition from user to Trusted Committer. It is also a
```

Comment:
Just out of curiosity, it seems that TC status would be something that would be useful for people to display as a badge. Has anyone used badges in this fashion?